reaper-mouse.ini

```diff
@@ -52,3 +52,6 @@ mm_4=2
 [MM_CTX_RULER_DBLCLK]
 mm_0=_BR_PLAY_MOUSECURSOR
 
+[MM_CTX_TRACK_DBLCLK]
```

Comment:
Was bedeutet die Zeile? Was macht "track" anders als "ruler"?

Reply:
Ja, die Dokusituation für Reaper ist echt mau, aber ich habe die reaper-mouse.ini mal dokumentiert:

http://mespotine.de/Ultraschall/Reaper_Filetype_Descriptions.html#Reaper-mouse.ini

Der Unterschied zwischen RULER und TRACK ist, dass RULER nur für die Zeitleiste oben ist, während TRACK für die einzelnen Tracks da ist.
Auf diese Art kannst Du entscheiden, was bei nem Doppelklick im Ruler passieren soll. Und wenn Du was Anderes für Doppelclick im Track haben willst, stellste halt was Anderes ein.

Reply:
Danke für die Infos.

Das ist ja äußerst bescheiden. Das wird ja gigantisch, bei einer solch ausführlich dokumentierten Software neue Features zu implementieren..... :-/

Reply:
Na wie in meinem Video gezeigt - Repo lokal klonen, die REAPER.app oder .exe hinzu legen und starten.

Und das mit der Doku, ja ist nicht toll, aber hey wir machen den Job seit vier Jahren :-) Das Forum ist hier noch oft hilfreich:
https://forum.cockos.com/forumdisplay.php?f=20

Reply:
Ok, Punkt für dich. Das Video habe ich nicht zu Ende gesehen.
Ich muss mir noch überlegen, wo ich mich am sinnvollsten (und für mich am wenigsten schmerzhaften) einbringen kann (und will ;-).
Doku ist auf jeden Fall ein Punkt. Jetzt geh ich erst mal Video gucken.